http/src/main/resources/reference.conf

```diff
@@ -67,10 +83,17 @@ gce {
       creating = 30 minutes
       starting = 20 minutes
       deleting = 30 minutes
+      stopping = 30 minutes
```

Comment:
What's the behavior if a runtime times out in `Stopping` status? I think I avoided adding a timeout because I didn't want Leo to auto-delete the runtime. But probably good to at least alert if this happens.

Reply:
ah good point..I think I had a comment somewhere wondering if this is safe to do...maybe I'll treat stopping time out differently if it's easy to do. Otherwise I might just leave this out for now since the reason I did this is because I saw clusters stuck in `Stopping` in perf tests a few times, but I think I finally understand what happened https://broadinstitute.slack.com/archives/GB1VAM14J/p1588953652207800

Reply:
removing the timeout for now just to introduce less changes in this PR